google-cloud-translate-v2/Rakefile

```diff
@@ -0,0 +1,187 @@
+# frozen_string_literal: true
```

Comment:
Do you want `frozen_string_literal: true` in these files? It appears in some of them but not most.

Reply:
Removed.